documentation/src/main/asciidoc/topics/ref_server_realm_trust.adoc

```diff
@@ -1,18 +1,26 @@
-[id='trust_store_realm-{context}']
+[id='trust-store-realms_{context}']
 = Trust Store Realms
-Trust store realms use keystores that contain certificates, or certificate chains, that clients must present to connect to {brandname} Server.
+{brandname} Server can use Java keystores to that contain server and client certificates that encrypt traffic to and from endpoints.
```

Comment:
"{brandname} Server can use keystores containing server and client certificates in order to encrypt endpoint traffic."